### PR TITLE
Add documentation for comment.js

### LINKED ISSUE #29: Document wp-admin/js/comment.js

### PR DESCRIPTION
**Pull request by IreneStr**
Fixes #29

### CODE REVIEW

File: src/wp-admin/js/comment.js [hunk 1]
```diff
@@ -1,4 +1,12 @@
 /* global postboxes, commentL10n */
+
+/**
+ * @summary Binds to the document ready event.
+ *
+ * @since 2.5.0
+ *
+ * @param {jQuery} $ The jQuery object.
```
Comment (boblinthorst) [inline src/wp-admin/js/comment.js]: Add "@returns {void}"?

File: src/wp-admin/js/comment.js [hunk 2]
```diff
@@ -9,8 +17,16 @@ jQuery(document).ready( function($) {
...
+	/**
+	 * @summary Adds event that opens the time stamp form if the form is hidden.
+	 *
+	 * @listens $edittimestamp:click
+	 *
+	 * @param {Event} event The event object.
+	 */
 	$edittimestamp.click( function( event ) {
 		if ( $timestampdiv.is( ':hidden' ) ) {
+			// Slide down the from and sets focus on the first field.
```
Comment (boblinthorst) [inline src/wp-admin/js/comment.js]: **form**

File: src/wp-admin/js/comment.js [hunk 3]
```diff
@@ -32,6 +56,13 @@ jQuery(document).ready( function($) {
 		event.preventDefault();
 	});
 
+	/**
+	 * @summary Sets the time stamp values when the ok button is clicked.
```
Comment (boblinthorst) [inline src/wp-admin/js/comment.js]: Probably not a problem, but inconsistent description with the listener above this one. 
first one: "when 'cancel' is clicked".
second one: "when the ok button is clicked".

**Comment by thulshof** [comment]
CR2 done 👍

**Comment by moorscode** [comment]
Patch created https://core.trac.wordpress.org/ticket/41065

**Comment by omarreiss** [comment]
Patch accepted

**Pull request closed without merging**